### ISSUE TITLE
Mandrel release doesn't contain proper *.src.zip files

### ISSUE THREAD

**Issue by jerboaa** [graalvm/mandrel#54]
A mandrel release from here:
https://github.com/graalvm/mandrel/releases/tag/mandrel-20.1.0.0.Alpha1

... doesn't contain proper `*.src.zip` files for graal vm jars. Including those `.src.zip` files improves debugging experience of native images where the debug info generator produces `sources` folder with `jdk`, `graal`, `src` folders if corresponding sources of the app are being found. In particular, I don't get `sources/graal` folder for this release only `sources/jdk` for a native image generated with:

```
native-image -H:GenerateDebugInfo=1 ...
```

Actual:

```
$ ls sources/
jdk
```

Expected:

```
$ ls sources/
jdk graal
```

**Comment by zakkak** [comment]
@jerboaa can you please see if https://github.com/mandrel/mandrel-packaging/pull/27 fixes the issue for you?

Using it I see:
```
$ tree -L 4 sources
sources
├── graal
│   ├── com
│   │   └── oracle
│   │       └── svm
│   └── org
│       └── graalvm
│           ├── collections
│           ├── compiler
│           ├── nativeimage
│           └── util
├── jdk
│   ├── java.base
│   │   ├── java
│   │   │   ├── io
│   │   │   ├── lang
│   │   │   ├── math
│   │   │   ├── nio
│   │   │   ├── security
│   │   │   ├── text
│   │   │   ├── time
│   │   │   └── util
│   │   ├── jdk
│   │   │   └── internal
│   │   └── sun
│   │       ├── nio
│   │       ├── text
│   │       └── util
│   ├── jdk.localedata
│   │   └── sun
│   │       └── util
│   └── jdk.unsupported
│       └── sun
│           └── misc
└── src
    └── HelloStrict.java
```

**Comment by zakkak** [comment]
Closed by https://github.com/mandrel/mandrel-packaging/pull/27

**Comment by zakkak** [comment]
Hmmm, sorry. This should stay open till a new release with the src files gets released.

**Comment by zakkak** [comment]
We are planning a new release in the following weeks which will include the src files.

**Comment by zakkak** [comment]
Sources are included since 20.1.0.1.Alpha2
